@xen-orchestra/lite/src/components/RemoteConsole.vue

```diff
@@ -61,9 +61,7 @@ const clearVncClient = () => {
   vncClient.removeEventListener('disconnect', handleDisconnectionEvent)
   vncClient.removeEventListener('connect', handleConnectionEvent)
 
-  if (vncClient._rfbConnectionState !== 'disconnected') {
-    vncClient.disconnect()
-  }
```

Comment:
To explain the reason for this change: we discussed and agreed that we should not rely on "private" internal state.

It's also the reason why `_rfbConnectionState` is not available in the type definitions.

I checked the source code, and the only thing that happens when you try to `disconnect()` being already disconnected is a message in the console.